<launch>
    <node name="joy_node" pkg="joy" type="joy_node" output="screen"/>
    <node name="teleop_js" pkg="Navigation" type="teleop_js" output="screen"/> 
</launch>
